about.html

```diff
@@ -0,0 +1,125 @@
+      </ul>
+    </div>
+      <header class="header">
+          <div class="headerelement headerelement1"></div>
+          <nav class="headerelement headerelement2">
+            <div class="lines">
+              <div class="menuline line-1"></div>
+              <div class="menuline line-2"></div>
+              <div class="menuline line-3"></div>
+            </div>
+            <ul class="menuitems">
+              <li class="menuitem menuitem-1">
+                  <a href="./index.html">Home</a>
+              </li>
+              <li class="menuitem menuitem-2">
+                <a href="./about.html">About</a>
+              </li>
+              <li class="menuitem menuitem-3">
+                <a href="#">lorem</a>
+              </li>
+              <li class="menuitem menuitem-4">
+                <a href="#">Ipsum</a>
+              </li>
+            </ul>
+            <div class="navmenubottom"></div>
+          </nav>
+          <nav class="header-desktop">
+            <div class="headerimgcontainer"><a href="./index.html"><img class="headerimg" src="./imgs/E3logowhite.png" alt=""></a></div>
+            <ul class="headercontainer">
+              <li class="header-desktop1"><a href="./index.html">Home</a></li>
+              <li class="header-desktop2"><a href="./about.html">About</a></li>
+              <li class="header-desktop3"><a href="#">Lorem</a></li>
+              <li class="header-desktop4"><a href="#">Ipsum</a></li>
+            </ul>
+            <div class="hbuttoncontainer"><button class="headerbutton">News</button></div>
+          </nav>
+        </header>
+      <main>
+        <div class="a-headline">
+          <p class="a-ph1">"Stay in the game!"</p>
+          <h1  class="a-p1">
+            <span>Worldwide broadcast E3 2021
+          </h1>
+          <h2>
+            <span class="a-pd1">Worldwide broadcast E3 2021
+          </h2>
+          <p class="a-pheadline">
+            E3 is the world’s premier event for computer and video games and related products. 
+            The show is owned and operated by the ESA, the US trade association dedicated to serving 
+            the business and public affairs needs of the companies developing and publishing 
+            interactive games for video game consoles, handheld devices, personal computers, and the 
+            internet.
+          </p>
+          <p class="a-contact">Press Contacts: Kat Jones, Motiv PR, kjones@motivpr.com
+            and Johner Riehl, Wonacott PR, jriehl@wonacottpr.com</p>
+        </div>
+        <section class="info">
+          <h2 class="infotittle">Electronic Entertainment Expo Logo 2021</h2>
+          <h2 class="eventsline">____</h2>
+          <p class="infodescpt">The E3 logo and variations are trademarks and/or registered trademarks 
+            of the Entertainment Software Association.</p>
+          <div class="infocard"> <img class="logoimg" src="./imgs/E3Logo.png" alt=""></div>
```

Comment:
- [ ] A good practice for `<img>` tag is always add a value for `alt` tag, please add it. 🐻